packages/host/src/Viewport.tsx

```diff
@@ -1,35 +1,28 @@
 import React from 'react';
-import { CssBaseline, makeStyles } from '@openmsupply-client/common';
-
-const useGlobalStyles = makeStyles(
-  () => ({
-    '@global': {
-      '*:-webkit-full-screen': {
-        height: '100%',
-        width: '100%',
-      },
-      html: {
-        position: 'fixed',
-      },
-      'html, body': {
-        height: '100%',
-        width: '100%',
-      },
-      '#root': {
-        height: '100%',
-        display: 'flex',
-        flexDirection: 'column',
-      },
-    },
-  }),
-  { name: 'ViewportGlobals' }
-);
+import { GlobalStyles, CssBaseline } from '@openmsupply-client/common';
 
 const Viewport: React.FC = props => {
-  useGlobalStyles();
-
   return (
     <React.Fragment>
+      <GlobalStyles
+        styles={{
+          '*:-webkit-full-screen': {
+            height: '100%',
+            width: '100%',
+          },
+          '#root': {
+            height: '100%',
+            display: 'flex',
+            flexDirection: 'column',
+          },
+          html: { position: 'fixed' },
+          'html, body': {
+            height: '100%',
+            width: '100%',
+          },
+        }}
+        {...props}
+      />
```

Comment:
I had made some other changes and tried a different way, but I didn't like it so I reverted it. That way had used this `GlobalStyles` instead of `makeStyles` so I left it like that

Reply:
can it be `styled` rather than adding the styles prop?

Reply:
I tried, but it can't be. `styled` makes a `className` and applies it to the component, where as this seems to need this object passed through put in a var instead.